index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Cart App</title>
+  </head>
+  <body>
+    <!---CART-->
+    <div class="cart">
+        <p class="cart-title">Books Cart</p>
+        <!---TITLE OR HEADER OF THE CART-->
+        <div class="cart-header">
+            <h3>Book Name</h3>
+            <h3>Price</h3>
+            <h3>Quantity</h3>
+            <h3> Edit  </h3>
+        </div>
+        <!---CONTENTS OF THE CART-->
+        <div class="cart-item">
+            <div class="cart-product-titke">Thinking fast and slow</div>
+            <div class="cart-price">$30.00</div>
+            <input type="number" value="1" class="cart-quantity">
+            <!--REMOVE FROM CART-->
+            <div id="actions-on-cart">
+                <i class="fa fa-trash trash-btn" ></i> 
+                <i class="fa fa-heart like-btn notLiked"></i> 
+
+            </div>
+            
+            
+        </div>
+        <div class="cart-item">
+            <div class="cart-product-titke">Thinking fast and slow</div>
+            <div class="cart-price">$30.00</div>
+            <input type="number" value="1" class="cart-quantity">
+            <!--REMOVE FROM CART-->
+            <div id="actions-on-cart">
+                <i class="fa fa-trash trash-btn" ></i> 
+                <i class="fa fa-heart like-btn notLiked" ></i> 
```

Comment:
As mentioned in a comment above, the not liked state is the default so no need for the extra class.